AnkiDroid/src/main/java/com/ichi2/anki/AnkiActivity.java

```diff
@@ -91,14 +92,10 @@ protected void onCreate(Bundle savedInstanceState) {
         Themes.setTheme(this);
         super.onCreate(savedInstanceState);
         // Disable the notifications bar if running under the test monkey.
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {        if (AdaptionUtil.isUserATestClient()) {
-            getWindow().getDecorView().getWindowInsetsController().hide(android.view.WindowInsets.Type.statusBars());
-        }
-        else {
-            if (AdaptionUtil.isUserATestClient()) {
-                getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN);
-            }
-        }
+
+        if (AdaptionUtil.isUserATestClient()) {
+            Compat compat = CompatHelper.getCompat();
+            compat.setFullscreen(getWindow());
```

Comment:
Much nicer! Just a nitpick

```suggestion
            CompatHelper.getCompat().setFullscreen(getWindow();
```